images/pulp/Containerfile

```
@@ -0,0 +1,16 @@
+ARG PULPCORE_VERSION=nightly
+
+FROM quay.io/centos/centos:stream9
+
+ARG PULPCORE_VERSION
+
+RUN dnf -y update && \
+    dnf -y --nodocs --setopt install_weak_deps=False install \
+    https://yum.theforeman.org/pulpcore/${PULPCORE_VERSION}/el9/x86_64/pulpcore-repos-${PULPCORE_VERSION}-0.1.nightly.el9.noarch.rpm && \
+    dnf -y --nodocs --setopt install_weak_deps=False install \
+    pulpcore \
+    "pulpcore-plugin(rpm)" \
+    "pulpcore-plugin(ostree)" \
+    "pulpcore-plugin(container)" \
+    "pulpcore-plugin(python)" && \
```

Thoughts on:
```suggestion
    pulpcore pulpcore-plugin\({rpm,ostree,container,python}\) && \
```

It may be too smart, but may also be a path to making it an easy arg to add so you can easily configure it without changing the `Containerfile` itself.